packages/tooltip/src/Tooltip.stories.tsx

```diff
@@ -45,3 +45,9 @@ export const Icon: Story<TooltipProps> = (props) => (
     <Question />
   </Tooltip>
 )
+
+export const KeepWhileHovering: Story<TooltipProps> = (props) => (
+  <Tooltip keepWhileHovering {...props}>
```

Comment:
Why not make it default behavior? E.g. in ant design it's persistent by default https://ant.design/components/tooltip

Reply:
My opinion is yes, we can do it. It was @DiRaiks asking to make it as prop, we should ask him why he wanted it to be parameterizable.

Reply:
I think that this behavior is not standard for a tooltip. But can we change the parameter name to something simpler? For example, as attributes of html tags.
@kozlovzxc-at-lido @manneredboor wdyt?
Or do you think this behavior should become permanent?

Reply:
I do not see a ux problem to make it permanent, maybe I am missing cases which can be problematic.

Reply:
@DiRaiks @kozlovzxc-at-lido, I have done it in this way according our discussion in discord